packages/global-cli/index.js

```diff
@@ -36,15 +36,12 @@
 // /!\ DO NOT MODIFY THIS FILE /!\
 // ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-'use strict';
```

Comment:
removing `'use strict'` requires adding `@babel/plugin-transform-strict-mode` to babel config.

Reply:
Gotcha. Going to update that.

Reply:
ESLint configuration is set to assume we are dealing with ES6 modules (implied strict mode as per spec) where in fact, we are not. Gotta change that and Babel plugin is a good one too.